### PR TITLE
IBTA-52. View Employees

### PR DESCRIPTION
**Pull request by devJUKI**

### CODE REVIEW

File: src/CSI.IBTA.Employer/Clients/EmployeesClient.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using CSI.IBTA.Employer.Endpoints;
+using CSI.IBTA.Employer.Interfaces;
+using CSI.IBTA.Shared.DTOs;
+using Newtonsoft.Json;
+
+namespace CSI.IBTA.Employer.Clients
+{
+    public class EmployeesClient : IEmployeeClient
```
Comment (radislavkireev) [inline src/CSI.IBTA.Employer/Clients/EmployeesClient.cs]: Make naming consistent. if class is in plural Employee**s**, the i-face should be plural too IEmployee**s**

Comment (radislavkireev) [inline src/CSI.IBTA.Employer/Clients/EmployeesClient.cs]:
```suggestion
    internal class EmployeesClient : IEmployeeClient
```

File: src/CSI.IBTA.Employer/Clients/AuthorizedHttpClient.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+﻿using System.Net.Http.Headers;
+
+namespace CSI.IBTA.Employer.Clients
+{
+    public class AuthorizedHttpClient : HttpClient
```
Comment (radislavkireev) [inline src/CSI.IBTA.Employer/Clients/AuthorizedHttpClient.cs]:
```suggestion
    internal class AuthorizedHttpClient : HttpClient
```

File: src/CSI.IBTA.UserService/Services/EmployeesService.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+﻿using CSI.IBTA.UserService.Interfaces;
+using CSI.IBTA.DataLayer.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using CSI.IBTA.Shared.Entities;
+using CSI.IBTA.Shared.DTOs;
+using AutoMapper;
+
+namespace CSI.IBTA.UserService.Services
+{
+    public class EmployeesService : IEmployeesService
```
Comment (radislavkireev) [inline src/CSI.IBTA.UserService/Services/EmployeesService.cs]:
```suggestion
    internal class EmployeesService : IEmployeesService
```

**Review by radislavkireev — APPROVED** [review]

**Review by VidasBui — APPROVED** [review]

**Pull request merged**